static/js/request_row.js

```diff
@@ -1,7 +1,6 @@
 import $ from 'jquery';
 import moment from 'moment';
 import {datetimeFormat} from './utils.js';
-import tooltip from 'bootstrap';
```

Comment:
I don't know why this was added in the observation-portal, but I couldn't see any difference after removing it and codify complains it's unused.

Reply:
Probably just a leftover from before that wasn't removed